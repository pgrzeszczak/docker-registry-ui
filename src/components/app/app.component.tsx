import * as React from 'react';
import AppBar from 'material-ui/AppBar';

export class App extends React.Component {
  render() {
    return (
      <AppBar
        title="Docker Registry UI"
      />
    );
  }
}
