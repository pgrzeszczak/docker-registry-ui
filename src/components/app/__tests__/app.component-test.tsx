import * as React from 'react';
import { App } from '../app.component';
import toJson from 'enzyme-to-json';
import * as Enzyme from 'enzyme';
import * as Adapter from 'enzyme-adapter-react-16';

import 'jest';

Enzyme.configure({ adapter: new Adapter() });

it('Should be rendered', () => {
  const component = Enzyme.shallow(
    <App />
  );
  expect(toJson(component)).toMatchSnapshot();
});
