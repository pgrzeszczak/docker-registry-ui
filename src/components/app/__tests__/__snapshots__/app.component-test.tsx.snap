// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should be rendered 1`] = `
<AppBar
  showMenuIconButton={true}
  title="Docker Registry UI"
  zDepth={1}
/>
`;
